readme.md

```diff
@@ -88,4 +97,42 @@ The metric that would actually measure the business outcome is transmission rate
 
 ### Icebox
 1. “Initiative2.epic3.story2” 
-2. “Initiative2.epic4” 
+2. “Initiative2.epic3.story3” 
+3. “Initiative2.epic4” 
+
+## <a name="#instructions">Instructions
+
+### Step 1. Clone the repository
+```
+git clone git@github.com:qinlang0628/2020-msia423-Qin-Covid19.git
+cd 2020-msia423-Qin-Covid19
+git checkout development
+```
+### Step 2. Build the docker
+```
+docker build -t runapp .
+```
+### Step 3. Download the data from [John Hopkins CSSE](https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_time_series/time_series_covid19_confirmed_global.csv) 
+```
+docker run --mount type=bind,source="$(pwd)"/data,target=/app/data runapp python3 src/download_data.py
+```
+By default, the data will be downloaded to data/sample, you can also specify the output path by changing the OUTPUT_PATH in ```src/config.py```.
+
+### Step 4. Upload the data to AWS S3.
+
+To connect to the AWS S3, you need to change the credential information in ```config/aws_s3.conf``` by replacing \<access key id> and \<secret access key> with your own key, which can be found in "security_credentials" section under your AWS account.
+```
+docker run --mount type=bind,source="$(pwd)"/data,target=/app/data runapp python3 src/upload_data.py
+```
+By default the bucket name is ```nw-langqin-s3```, you can also specify your own own bucket by add a ```--bucket <bucket name>``` or ```-b <bucket name>``` .
+
+### Step 5. Create an empty database
+- Choice 1: The database could be created locally by the command below:
+```
+docker run --mount type=bind,source="$(pwd)"/data,target=/app/data runapp python3 src/models.py
+```
+- Choice 2: If you want to create the database in AWS RDS, you need to modify the ```config/aws_rds.conf``` by replacing \<user>, \<password>, \<host>, \<port>, \<database> by your own information. Then type the command below:
+```
+docker run --mount type=bind,source="$(pwd)"/data,target=/app/data runapp python3 src/models.py --rds
+```
```

Comment:
Have to inform the user that the docker image needs to be rebuilt every time when AWS/RDS config file changes, which is not very efficient. (Can use environment file instead)

Reply:
The "rebuild" information has been added to the readme.md. It is a great idea to use the environment file, but due to the time constraint, the improvement will be made in the future.